tests/Install-Scripts.Test/GivenThatIWantToInstallDotnetFromAScript.cs

```diff
@@ -251,8 +245,7 @@ public void WhenInstallingDotnetRuntime(string channel, string? quality, string
         public void WhenInstallingAspNetCoreRuntime(string channel, string? quality, string versionRegex)
         {
             if (channel == "release/3.0"
-                || channel == "release/3.1"
-                || channel == "release/5.0")
+                || channel == "release/3.1")
```

Comment:
same here, probably better to comment it out in test data collections.

Reply:
That would disable it in all runtimes and requires separation of test cases for each runtime. I will leave this as is for now to not invest too much time in this area.